public/app/features/plugins/admin/hooks/useHistory.tsx

```diff
@@ -1,13 +1,9 @@
-import { getLocationSrv } from '@grafana/runtime';
+import { locationService } from '@grafana/runtime';
 
 export const useHistory = () => {
   return {
     push: ({ query }: any) => {
-      getLocationSrv().update({
-        partial: true,
-        replace: false,
-        query,
-      });
+      locationService.partial(query);
```

Comment:
Looking at this hook now, and it's usage in the plugins catalog, I think it's an abstraction too many. Maybe something for us to remove in a future PR?

Reply:
Agree with Jack on this one.

Reply:
Totally agree!